extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -49,6 +49,9 @@
     "pageview": "//webcare.byside.com/BWA!webcareId/amp/pixel.php?webcare_id=!webcareId&bwch=!channel&lang=pt&fid=!fid&bwit=A&tuid=_client_id_&suid=&puid=_page_view_id_p_timestamp_&referrer=_document_referrer_&page=_source_url_&amppage=_ampdoc_url_&bwpt=_title_&bres=_viewport_width_x_viewport_height_&res=_screen_width_x_screen_height_&v=v20171116a&ampv=_amp_version_&viewer=_viewer_&ua=_user_agent_&r=_random_",
     "event": "//webcare.byside.com/BWA!webcareId/amp/signal.php?event_id=!eventId&event_label=!eventLabel&fields=!fields&webcare_id=!webcareId&bwch=!channel&lang=pt&fid=!fid&bwit=A&tuid=_client_id_&suid=&puid=_page_view_id_p_timestamp_&referrer=_document_referrer_&page=_source_url_&amppage=_ampdoc_url_&bwpt=_title_&bres=_viewport_width_x_viewport_height_&res=_screen_width_x_screen_height_&v=v20171116a&ampv=_amp_version_&viewer=_viewer_&ua=_user_agent_&r=_random_"
   },
+  "captainmetrics": {
+    "hit": "https://photon.captainmetrics.com/amp/?pId=!projectId&cId=!deviceId&sId=!sessionId&ts=_timestamp_&rand=_random_&ampV=_amp_version_&v=1&lang=_browser_language_&ua=_user_agent_&res=_screen_height_x_screen_width_&aRes=_available_screen_height_x_available_screen_width_&off=_timezone_&tz=_timezone_code_&lp=_page_&ref=_external_referrer_&ampRef=_ampdoc_url_&pageId=_page_view_id_&type=timeSpent&secs=5"
```

Comment:
Typo, replace `pId` to `cId`. 
Please double check and fix the test. Thanks

Reply:
Just fixed the test, anyway the server-side only expects a POST payload to validate the data, any GET request (that the test does) will get a basic 200 response.